regtests/bin/matrix.base

```diff
@@ -2124,25 +2124,50 @@
  #Test of UFS applications with ww3_multi and grib2 output
   if [ "$ufs" = 'y' ] && [ "$grib" = 'y' ]
   then
-    echo ' ' >> matrix.body
-    echo "$rtst -w work_a -m grdset_a -f -p $mpi -n $npl -t $nth1  -o all $ww3 ww3_ufs1.1"                    >> matrix.body
-    if [ "$npl_b4b" = 'y' ]
-    then
-       echo "$rtst -w work_a_npl -m grdset_a -f -p $mpi -n $npl1 -t $nth1 -o all $ww3 ww3_ufs1.1"             >> matrix.body
-       echo "./bin/test.comp ww3_ufs1.1 work_a work_a_npl"                                                    >> matrix.body
-    fi
-    if [ "$nth_b4b" = 'y' ]
-    then
-       echo "$rtst -w work_a_nth -m grdset_a -f -p $mpi -n $npl -t $nth -o all $ww3 ww3_ufs1.1"               >> matrix.body
-       echo "./bin/test.comp ww3_ufs1.1 work_a work_a_nth"                                                    >> matrix.body
-    fi
-    if [ "$rstrt_b4b" = 'y' ]
-    then
-       echo "mkdir -p ww3_ufs1.1/work_b"                                                                      >> matrix.body
-       echo "cp ww3_ufs1.1/work_a/20210401.030000.restart.glo_1deg ww3_ufs1.1/work_b/restart.glo_1deg"        >> matrix.body
-       echo "$rtst -w work_b -m grdset_b -f -p $mpi -n $npl -t $nth1  -o all $ww3 ww3_ufs1.1"                 >> matrix.body
-       echo "./bin/test.comp ww3_ufs1.1 work_a work_b"                                                        >> matrix.body
-    fi
+     # (light grid for single core/laptop check)
+     if [ "$ufscoarse" = 'y' ]
+     then
+       echo ' ' >> matrix.body
+       echo "$rtst -w work_c -m grdset_c -f -p $mpi -n $np  -o all $ww3 ww3_ufs1.1"                           >> matrix.body
+        if [ "$npl_b4b" = 'y' ]
+        then
+           halfnp=$(($np / 2))
+           echo "$rtst -w work_c_npl -m grdset_c -f -p $mpi -n $halfnp -o all $ww3 ww3_ufs1.1"                >> matrix.body
+           echo "./bin/test.comp ww3_ufs1.1 work_c work_c_npl"                                                >> matrix.body
+        fi
+        if [ "$nth_b4b" = 'y' ]
+        then
+           echo "$rtst -w work_c_nth -m grdset_c -f -p $mpi -n $nr -t $nth -o all $ww3 ww3_ufs1.1"            >> matrix.body
```

Comment:
Need a new switch for this, the existing switch does not have OMPG and OMGH flags (as they are in mode/esmf/switch).  Not sure if this is the case for the other ufs tests as well or not.